massgov/mayflower
# DP-9576 Mayflower: Remove "raw" filter from 02-molecules/listing-table.twig

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

Remove raw filter from patternlab/styleguide/source/_patterns/02-molecules/listing-table.twig.

Use of the template in Mayflower:
```
 ~/MassIT/mayflower/ [patternlab/DP-9576_molecules-listing-table] grep -r 'listing-table.twig' patternlab/styleguide/source/_patterns
patternlab/styleguide/source/_patterns/03-organisms/by-template/page-header~for-court-rules.json:        "path": "@molecules/listing-table.twig",
patternlab/styleguide/source/_patterns/04-templates/01-content-types/binder-page.twig:          {% include "@molecules/listing-table.twig" %}
patternlab/styleguide/source/_patterns/04-templates/01-content-types/court-rules.json:        "path": "@molecules/listing-table.twig",
patternlab/styleguide/source/_patterns/04-templates/01-content-types/court-rules.twig:          {% include "@molecules/listing-table.twig" %}
patternlab/styleguide/source/_patterns/04-templates/01-content-types/policy-advisory.json:      "path": "@molecules/listing-table.twig",
patternlab/styleguide/source/_patterns/05-pages/binder-page-internal.json:      "path": "@molecules/listing-table.twig",
patternlab/styleguide/source/_patterns/05-pages/board-decision.json:      "path": "@molecules/listing-table.twig",
patternlab/styleguide/source/_patterns/05-pages/executive-order.twig:  {% include "@molecules/listing-table.twig" %}
patternlab/styleguide/source/_patterns/05-pages/information-details.twig:    {% include "@molecules/listing-table.twig" %}
patternlab/styleguide/source/_patterns/05-pages/policy-advisory-directive.json:      "path": "@molecules/listing-table.twig",
patternlab/styleguide/source/_patterns/05-pages/regulation.json:      "path": "@molecules/listing-table.twig",
patternlab/styleguide/source/_patterns/05-pages/rules-of-court.json:        "path": "@molecules/listing-table.twig",
 ~/MassIT/mayflower/ [patternlab/DP-9576_molecules-listing-table]
```

## Related Issue / Ticket

- [DP-9576 Mayflower: Remove "raw" filter from 02-molecules/listing-table.twig](https://jira.mass.gov/browse/DP-9576)

## Steps to Test


1.  Go to the following pages and check for any display issue with the listing table:

- advisory page
- binder page
- decision page
- executive order page
- information details page
- regulation page
- rules of court page

## Screenshots
**binder page**
<img width="496" alt="dp-9576_table-binder-cover" src="https://user-images.githubusercontent.com/9633303/43420797-df464838-9412-11e8-8367-65b2155bf597.png">

**decision page**
<img width="642" alt="dp-9576_table-decision" src="https://user-images.githubusercontent.com/9633303/43420802-e5656046-9412-11e8-9424-c7e08248ef4f.png">

**executive order page**
<img width="594" alt="dp-9576_table-executive-order" src="https://user-images.githubusercontent.com/9633303/43420809-edc61578-9412-11e8-99f3-65ce00f4f2ec.png">

**advisory page**
<img width="558" alt="dp-9576_table-advisory" src="https://user-images.githubusercontent.com/9633303/43420818-f2a58a42-9412-11e8-9fd3-acc48bcf6bd4.png">

**regulation page**
<img width="603" alt="dp-9576_table-regulation" src="https://user-images.githubusercontent.com/9633303/43420824-f941c0be-9412-11e8-852b-b00134d38e61.png">

**rules of court page**
<img width="487" alt="dp-9576_table-rules-of-court" src="https://user-images.githubusercontent.com/9633303/43420839-ff31b088-9412-11e8-8294-546aab11fdd0.png">

**information details page**
<img width="446" alt="dp-9576_table-info-details" src="https://user-images.githubusercontent.com/9633303/43420844-040fca4a-9413-11e8-9d85-03c51f556ccd.png">


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
